kayenta-stackdriver/src/test/groovy/com/netflix/kayenta/providers/metrics/StackdriverCanaryMetricSetQueryConfigSpec.groovy

```diff
@@ -54,23 +54,23 @@ class StackdriverCanaryMetricSetQueryConfigSpec extends Specification {
     given:
     CanaryConfig canaryConfig = CanaryConfig.builder().templates(templates).build()
     StackdriverCanaryMetricSetQueryConfig stackdriverCanaryMetricSetQueryConfig =
-      StackdriverCanaryMetricSetQueryConfig.builder().customFilterTemplate(customFilterTemplate).customFilter(customFilter).build()
+      StackdriverCanaryMetricSetQueryConfig.builder().customFilterTemplate(customFilterTemplate).customInlineTemplate(customInlineTemplate).build()
     StackdriverCanaryScope stackdriverCanaryScope = new StackdriverCanaryScope(extendedScopeParams: scopeParams)
 
     expect:
     QueryConfigUtils.expandCustomFilter(canaryConfig, stackdriverCanaryMetricSetQueryConfig, stackdriverCanaryScope) == expectedExpandedTemplate
 
     where:
-    templates                                             | customFilterTemplate | customFilter          | scopeParams       || expectedExpandedTemplate
-    ["my-template": 'A test: key1=${key1}.']              | "my-template"        | "An explicit filter." | [key1: "value-1"] || "An explicit filter."
+    templates                                             | customFilterTemplate | customInlineTemplate          | scopeParams       || expectedExpandedTemplate
+    ["my-template": 'A test: key1=${key1}.']              | "my-template"        | "An inline template."         | [key1: "value-1"] || "An inline template."
```

Comment:
Since the `customInlineTemplate` values will now be expanded, should we include a couple of tests that have variables to be bound?